components/selection/selection-action-menu-item.js

```diff
@@ -23,11 +23,7 @@ class ActionMenuItem extends SelectionActionMixin(MenuItemMixin(LitElement)) {
 				}
 				d2l-icon {
 					flex: none;
-					margin-left: 6px;
-				}
-				:host([dir="rtl"]) d2l-icon {
```

Comment:
Another example where the RTL styles were not being applied due to the missing mixin.